### PR TITLE
fix wrong `navigator` property and update user agent string

### PR DESCRIPTION
**Pull request by T34-active**
### Related issues and pull requests

https://github.com/mdn/content/pull/32987

### CODE REVIEW

File: files/en-us/glossary/user_agent/index.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ Besides a browser, a user agent could be a bot scraping webpages, a download man
...
+A typical user agent string looks like this: `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0"`.
 
 ## See also
 
 - [User agent](https://en.wikipedia.org/wiki/User_agent) on Wikipedia
 - {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}}
 - [Browser detection using the user agent](/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent)
 - {{RFC(2616, "", "14.43")}}: The `User-Agent` header
-- [Glossary:](/en-US/docs/Glossary)
+- [Glossary](/en-US/docs/Glossary) :
```
Comment (skyclouds2001) [inline files/en-us/glossary/user_agent/index.md:25]:
```suggestion
- [Glossary](/en-US/docs/Glossary):
```

File: files/en-us/glossary/user_agent/index.md [hunk 2]
```diff
@@ -12,17 +12,17 @@ Besides a browser, a user agent could be a bot scraping webpages, a download man
 
 Spam bots, download managers, and some browsers often send a fake UA string to announce themselves as a different client. This is known as _user agent spoofing_.
 
-The user agent string can be accessed with {{Glossary("JavaScript")}} on the client side using the {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}} property.
+The user agent string can be accessed with {{Glossary("JavaScript")}} on the client side using the {{domxref("Navigator/userAgent", "navigator.userAgent")}} property.
```
Comment (skyclouds2001) [inline files/en-us/glossary/user_agent/index.md:15]:
```suggestion
The user agent string can be accessed with {{Glossary("JavaScript")}} on the client side using the {{domxref("navigator.userAgent")}} property.
```

File: files/en-us/glossary/user_agent/index.md [hunk 3]
```diff
@@ -12,17 +12,17 @@ Besides a browser, a user agent could be a bot scraping webpages, a download man
...
 ## See also
 
 - [User agent](https://en.wikipedia.org/wiki/User_agent) on Wikipedia
 - {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}}
 - [Browser detection using the user agent](/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent)
 - {{RFC(2616, "", "14.43")}}: The `User-Agent` header
-- [Glossary:](/en-US/docs/Glossary)
+- [Glossary](/en-US/docs/Glossary):
 
   - {{Glossary("Browser")}}
```
Comment (hamishwillee) [inline files/en-us/glossary/user_agent/index.md:27]:
```suggestion
- {{Glossary("Browser")}} (glossary)
```

File: files/en-us/glossary/user_agent/index.md [hunk 4]
```diff
@@ -12,17 +12,17 @@ Besides a browser, a user agent could be a bot scraping webpages, a download man
...
-The user agent string can be accessed with {{Glossary("JavaScript")}} on the client side using the {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}} property.
+The user agent string can be accessed with {{Glossary("JavaScript")}} on the client side using the {{domxref("navigator.userAgent")}} property.
 
-A typical user agent string looks like this: `"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:35.0) Gecko/20100101 Firefox/35.0"`.
+A typical user agent string looks like this: `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0"`.
 
 ## See also
 
 - [User agent](https://en.wikipedia.org/wiki/User_agent) on Wikipedia
 - {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}}
```
Comment (hamishwillee) [inline files/en-us/glossary/user_agent/index.md:22]:
```suggestion
{{domxref("navigator.userAgent")}}
```

File: files/en-us/glossary/user_agent/index.md [hunk 5]
```diff
@@ -12,19 +12,17 @@ Besides a browser, a user agent could be a bot scraping webpages, a download man
...
-- {{domxref("Navigator/userAgent", "NavigatorID.userAgent")}}
+{{domxref("navigator.userAgent")}}
 - [Browser detection using the user agent](/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent)
 - {{RFC(2616, "", "14.43")}}: The `User-Agent` header
-- [Glossary:](/en-US/docs/Glossary)
-
-  - {{Glossary("Browser")}}
+- {{Glossary("Browser")}} (glossary)
 
 - HTTP Headers
```
Comment (hamishwillee) [inline files/en-us/glossary/user_agent/index.md:27]:
```suggestion
- {{Glossary("Browser")}} (glossary)
- HTTP Headers
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much!

**Pull request merged**